src/chainparams.cpp

```diff
@@ -121,8 +121,11 @@ class CMainParams : public CChainParams {
         // This is fine at runtime as we'll fall back to using them as a oneshot if they don't support the
         // service bits we want, but we should get them updated to support all service bits wanted by any
         // release ASAP to avoid it where possible.
-        vSeeds.emplace_back("118.189.201.104");
+
         vSeeds.emplace_back("node2.walletbuilders.com");
+        vSeeds.emplace_back("118.189.201.104");
+        vSeeds.emplace_back("seed01.altcoinbuilders.com");
+        vSeeds.emplace_back("seed02.altcoinbuilders.com");
```

Comment:
Are these nodes active?

Reply:
> Are these nodes active?

Yes active at the moment and for a long period of time.